CHANGELOG.md

```diff
@@ -94,6 +94,11 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
   Contributed by @Conaclos
 
+- Added two new options to customise the emitted output of the CLI: `--json` and `--json-pretty`. With `--json`, the diagnostics and the
+  summary will be printed in the **terminal** in JSON format. Using `--json-pretty` _together_ with `--json` will output the same output, but formatted.
```

Comment:
Why do they need to be combined? The fact it requires so much emphasis seems to recognize it's a potential gotcha. Can't we make it more friendly by allowing `--json-pretty` to be used on its own?

Reply:
I thought about it, but this combination would require a check if both are passed and raise an error if they are.

I'm open to better names or options :)

Reply:
If both are specified, we can just let `--json-pretty` win, no?